utils/utils.ts

```diff
@@ -0,0 +1,44 @@
+import { createHmac } from 'crypto';
+import type { NextApiRequest } from 'next';
+import * as ethers from 'ethers';
+import fetch from 'node-fetch-retry';
+import Redis from 'ioredis';
+
+const fetchOptions = {
+    retry: 12,
+    pause: 1000,
```

Comment:
does this mean a single request could take up to 12 seconds?

Reply:
yes, it could be running for 12 seconds if it keeps not getting a response from etherscan